### PR TITLE
Keep up with the times

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**
Just some house cleaning.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -43,5 +43,5 @@ outputs:
   rebar3-version:
     description: Exact version of rebar3 that was installed
 runs:
-  using: node12
+  using: node16
```
Comment (paulo-ferraz-oliveira) [inline action.yml:46]: 16 is now the default, I think since Nov last year.

File: package.json [hunk 2]
```diff
@@ -19,24 +19,24 @@
...
+    "markdownlint": "0.25.1",
+    "prettier": "2.6.2",
     "shellcheck": "1.1.0",
     "yaml-lint": "1.2.4",
-    "yarn": "1.22.17"
+    "yarn": "1.22.18"
   },
   "engines": {
-    "node": ">=14"
+    "node": ">=16"
```
Comment (paulo-ferraz-oliveira) [inline package.json:40]: 16 is now the default, I think since Nov last year.

File: package.json [hunk 3]
```diff
@@ -19,24 +19,24 @@
   },
```
Comment (paulo-ferraz-oliveira) [inline package.json:19]: These were bumped to their latest. None of them appears to have broken interface.

File: src/setup-beam.js [hunk 4]
```diff
@@ -471,9 +471,11 @@ async function get(url0, pageIdxs) {
 
 function maybePrependWithV(versionToPrepend, specVersion) {
   const digitStart = /^\d+/
-  return digitStart.test(specVersion)
-    ? `v${versionToPrepend}`
-    : versionToPrepend
+  let v = versionToPrepend
```
Comment (paulo-ferraz-oliveira) [inline src/setup-beam.js:474]: I preferred the way it was, but I couldn't get the linter and formatter to work side-by-side. It didn't seem like a very bad compromise, this change.

**Review by starbelly — APPROVED** [review]

**Pull request merged**